data/game_config.cfg

```diff
@@ -106,6 +106,7 @@
         game_user_leave = "leave.wav"
         ready_for_start = "bell.wav"
         game_has_begun = "gamestart.ogg"
+		game_created = "chat-highlight.ogg"
```

Comment:
chat-highlight.ogg is a too high-urgency, and can be played for its usual meaning in the lobby (it sounds mild, but it's significance is that someone is directly asking you something).

Reply:
I didn't choose it, I'm just following what's specified in [game_config.cpp](https://github.com/wesnoth/wesnoth/pull/4259/files#diff-e7f34278f8d25cba252bd50a3e312654R265).

This file seems to override whatever is in the cpp file, though - I originally tested it with arrive.wav.

Reply:
I can change it if you want - what do you prefer?

Reply:
Using join.wav feels good to me, as I know from context that it's a new game rather than someone joining my game. But it may feel too high-urgency for other people.

Reply:
Done.